src/main/drivers/camera_control.c

```diff
@@ -105,15 +106,22 @@ static void cameraControlLo(void)
 void TIM6_DAC_IRQHandler(void)
 {
     cameraControlHi();
-
+#ifdef AT32F435
```

Comment:
Should we have a separate file, i.e. src/main/drivers/at32/camera_control.c ?

Reply:
would such introduce code-duplication? or can duplication be avoided?

Reply:
Yes, if we would extract it into a separate file, it's basically 90% code duplication.

For now this works best imho.

Reply:
Yes, I agree with @freasy. If we don't pay enough attention in the future these at32 drivers might became out of sync. But I would like to hear @haslinghuis opinion.

Reply:
![image](https://github.com/betaflight/betaflight/assets/8344830/7dfc7ce9-2dba-4bb5-8bf7-2d3dcb495cd4)

Reply:
Can you explain the reason? I'm not as experienced in C as you guys.

Reply:
You should ask @blckmn

Reply:
Split it into 3 files. One is common. The other two specific implementation for the respective chipset.